operations/patch_module.go

```diff
@@ -84,7 +84,7 @@ func PatchSetModule() cli.Command {
 			if err != nil {
 				return errors.Wrapf(err, "getting patch '%s'", patchID)
 			}
-			if existingPatch.IsCommitQueuePatch() {
+			if existingPatch.IsMergeQueuePatch() {
 				return errors.New("Use `commit-queue set-module` instead of `set-module` for commit queue patches")
```

Comment:
For my own understanding, does the `evergreen commit-queue set-module` workflow still apply for merge queue patches? It looks like this check was originally just for legacy commit queue patches.